index.js

```diff
@@ -77,15 +79,25 @@ function storeOptionValue(parseOptions, option, value, result) {
   }
 }
 
-const parseArgs = (
+const parseArgs = ({
   argv = getMainArgs(),
   options = {}
-) => {
+} = {}) => {
   validateArray(argv, 'argv');
   validateObject(options, 'options');
-  for (const key of ['withValue', 'multiples']) {
-    if (ObjectHasOwn(options, key)) {
-      validateArray(options[key], `options.${key}`);
+  for (const [option, optionConfig] of ObjectEntries(options)) {
+    validateObject(optionConfig, `options.${option}`);
+
+    if (ObjectHasOwn(optionConfig, 'type')) {
+      validateUnion(optionConfig.type, `options.${option}.type`, ['string', 'boolean']);
+    }
+
+    if (ObjectHasOwn(optionConfig, 'short')) {
+      validateString(optionConfig.short, `options.${option}.short`);
```

Comment:
Can/should we validate this is a single character?

Reply:
Yes.

Reply:
https://github.com/pkgjs/parseargs/pull/63/commits/b0b71c43ca10677ac9048cb9f7b1eff2c1d35987